misc/working_w_l10n_agency.md

```diff
@@ -0,0 +1,109 @@
+# Working with a Localization Agency
+
+This document is a guideline for project manager who interacts with a localization agency that is already vetted.
```

Comment:
@flodolo Would this work? `This document is a guideline for project manager who interacts with a localization agency that already successfully went through our [vetting process](https://docs.google.com/document/d/1Y0xfGo-24ZJI_uYFLtJFcuxCi20FNIqaeEc7v_FlaY4/edit). Our vetting process helps us ensure the vendors we work with are able to deliver translations in a timely manner, with the required level of quality and following our localization processes. We are also assessing their flexibility to work efficiently with our communities and our tools when it’s required.`